src/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore, applyMiddleware, combineReducers, compose } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+
+import './styles/index.css';
+import AppContainer from './containers/AppContainer';
+import registerServiceWorker from './registerServiceWorker';
+import recipeReducers from './store/recipe.reducer';
+
+const rootReducer = combineReducers({ recipes: recipeReducers });
+
+const store = createStore(
+  rootReducer,
+  {},
+  compose(applyMiddleware(thunk)),
+  window.__REDUX_DEVTOOLS_EXTENSION__&& window.__REDUX_DEVTOOLS_EXTENSION__()
```

Comment:
It looks like you followed the instructions for a basic store from the docs. Since you have middleware, look at section 1.2:
```
  import { createStore, applyMiddleware, compose } from 'redux';

+ const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+ const store = createStore(reducer, /* preloadedState, */ composeEnhancers(
- const store = createStore(reducer, /* preloadedState, */ compose(
    applyMiddleware(...middleware)
  ));
```